tests/test_wheel.py

```diff
@@ -239,3 +239,16 @@ def test_top_level_modules(package_module_types):
             'namespace',
             'native',
         }
+
+
+@pytest.mark.filterwarnings("ignore:::mesonpy")
+def test_install_subdir_python_path(wheel_install_subdir_python_path):
```

Comment:
You forgot to commit the test package. However, the test would be bogus, see my reply to #341